lib/v2/search.js

```diff
@@ -1,5 +1,18 @@
 const api = require('./api');
-const {isEmpty, isNumber} = require('../utils');
+const config = require('../config');
+const ensureOption = require('../utils/ensureOption').defaults(config());
+const {
+  isEmpty,
+  isNumber,
+  api_sign_request,
+  compact_and_sort,
+  compute_hash,
+  build_distribution_domain,
+  clear_blank
+} = require('../utils');
+const {DEFAULT_SIGNATURE_ALGORITHM} = require("../utils/consts");
+const {sign} = require("crypto");
```

Comment:
Why do we need this here?

Reply:
Typo, the require must've been by the IDE by mistake, good catch.